ros:
  bag: "/data/pointgrey/mono-4/calib_mono.bag"
  cam0_topic: "/camera/image_raw"

settings:
  data_path: "/data/pointgrey/mono-4"
  results_fpath: "/data/pointgrey/mono-4/calib_results.yaml"
  sigma_vision: 0.5
  imshow: true

calib_target:
  target_type: 'aprilgrid'  # Target type
  tag_rows: 6               # Number of rows
  tag_cols: 6               # Number of cols
  tag_size: 0.088           # Size of apriltag, edge to edge [m]
  tag_spacing: 0.3          # Ratio of space between tags to tagSize
                            # Example: tagSize=2m, spacing=0.5m --> tagSpacing=0.25[-]
cam0:
  proj_model: "pinhole"
  dist_model: "equi4"
  lens_hfov: 120
  lens_vfov: 120
  resolution: [1224, 1024]
  rate: 35.0
